app/controllers/foreman_ansible/concerns/hostgroups_controller_extensions.rb

```diff
@@ -18,6 +19,13 @@ def play_roles
 
       private
 
+      def check_hostgroup
+        return unless @hostgroup.hosts.empty?
```

Comment:
this is another place to be changed if we decide to run ansible on hosts assigned to children hostgroups as well but good for now